jest/helpers/asserts.js

```diff
@@ -1,5 +1,26 @@
+// ** SNAPSHOT FILTERING ***
+// prevent snapshots for logs that change on each instance, such as shuffled quiz options
+
+const excludedPhrases = [
+  "Cool! Here's what we get when we mix up the order of the answer choices:"
+]
+
+// returns true if log includes a string from the excludedPhrases array
+function isExcluded (log) {
+  for (let phrase of excludedPhrases) {
+    if (log.includes(phrase)) {
+      return true
+    }
+  }
+  return false
+}
```

Comment:
Just a suggestion to make it simpler:

```suggestion
function isExcluded (log) {
  return excludedPhrases.some(phrase => log.includes(phrase))
}
```